src/DssSpell.sol

```diff
@@ -37,12 +28,13 @@ contract DssSpellAction is DssAction {
         return false;
     }
 
-    // ----- USDP-PSM Facilitation Incentives -----
-    // Forum: https://forum.makerdao.com/t/usdp-psm-facilitation-incentives/22331
-    // Approve DAO Resolution hash QmWg43PNNGfEyXnTv1qN8dRXFJz5ZchrmZU8qH57Ki6D62
+    // ---------- HV Bank (RWA009-A) - Approve DAO Resolutions ----------
```

Comment:
The original instruction is not plural `HV Bank (RWA009-A) - Approve DAO Resolution`. Therefore I would keep it exactly as the original to avoid confusion

```diff
-    // ---------- HV Bank (RWA009-A) - Approve DAO Resolutions ----------
+    // ---------- HV Bank (RWA009-A) - Approve DAO Resolution ----------
```